kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/server/Server.kt

```diff
@@ -30,12 +30,16 @@ class Server(
       return
     }
 
-    val out = registry.invoke(
-      Context(jvmModule, routingClient),
-      verbRef.toModel(),
-      request.body.utf8()
-    )
-    response.onNext(CallResponse(body = out.encodeUtf8()))
+    try {
+      val out = registry.invoke(
+        Context(jvmModule, routingClient),
+        verbRef.toModel(),
+        request.body.utf8()
+      )
+      response.onNext(CallResponse(body = out.encodeUtf8()))
+    } catch (e: Throwable) {
+      response.onNext(CallResponse(error = CallResponse.Error(message = e.message ?: "internal error")))
```

Comment:
Will this include the stack trace? I think we'll want that, if it doesn't.

Reply:
added!